errors/index.js

```diff
@@ -10,6 +10,10 @@ exports.handlePsqlErrors = (err, req, res, next) => {
   }
   next(err);
 };
+
+exports.handleServerErrors = (err, req, res, next) => {
+  if (err.status === 500) res.status(500).send( { msg: 'Internal Server Error'})
+}
```

Comment:
For this one - you can simply hardcode a status of 500 without the conditional logic
If you even get to this error handler, it hasn't already been handled by one above.
Also, a great place to console.log the error as it's one you won't have come across yet!